src/components/text-input-with-copy-button/TextInputWithCopyButton.js

```diff
@@ -186,7 +188,7 @@ class TextInputWithCopyButton extends React.PureComponent<Props, State> {
         ) : null;
 
     render() {
-        const { className, ...rest } = this.props;
+        const { className, additionalButtons, ...rest } = this.props;
```

Comment:
this is super super minor but could we alphasort these?

```suggestion
        const { additionalButtons, className, ...rest } = this.props;
```

Reply:
good catch, thanks